docs/components/concepts/access-control/user-groups.md

```diff
@@ -0,0 +1,28 @@
+---
+id: user-groups
+title: "User groups"
+sidebar_label: "User groups"
+description: "Control the level of access a user or group has to perform tasks in the system via user task access restrictions."
+---
+
+User groups are a way to organize users in the system. They can be used to control the level of access a user or group has to perform tasks in the system via [user task restrictions](user-task-access-restrictions.md).
+
+### Creating a group
+
+To create a group, navigate to the **Organization** section of Console and click on the **Groups** tab.
+
+![Groups Management](../assets/access-control/group-management.png)
```

Comment:
Note that if a user hasn't created a group before, this screenshot will look different. I've adjusted the UI references below as such.